src/core/python2js.c

```diff
@@ -557,45 +525,134 @@ EM_JS_REF(JsRef, _JsArray_PostProcess_helper, (JsRef jscontext, JsRef array), {
     Hiwire.get_value(jscontext).dict_converter(Hiwire.get_value(array)));
 })
 
+// clang-format off
```

Comment:
Is there some reason to disable formatter here?

Reply:
Yeah, clang-format sometimes does annoying things to em_js. Try removing it and running clang-format and you'll see why I don't like it. The whole function body gets indented by 10 spaces or something like that.